be/src/olap/rowset/segment_v2/segment.h

```diff
@@ -74,7 +74,8 @@ class Segment : public std::enable_shared_from_this<Segment> {
 
     uint32_t num_rows() const { return _footer.num_rows(); }
 
-    Status new_column_iterator(const TabletColumn& tablet_column, ColumnIterator** iter);
+    Status new_column_iterator(const TabletColumn& tablet_column, const IOContext& io_ctx,
```

Comment:
Do not modify this method. could modify column iterator's init method and pass io ctx via init method.

Reply:
init is called just in new_column_iterator()